lib/configure.js

```diff
@@ -26,6 +26,10 @@ var readYaml = require('read-yaml');
 var travisEncrypt = promisify(require('travis-encrypt'));
 var writeYaml = require('write-yaml');
 
+// Uncomment this to record network requests/responses for automated tests.
+// var nock = require('nock');
+// nock.recorder.rec();
```

Comment:
> If we ever implement live tests with real GitHub repositories, we should enable this by default (when the test is running) to have more info when such a test fails.

Indeed. On the other hand, we can't enable this when running automated tests against the mocks, since enabling it disables those mocks. I've added some commentary to explain these use cases in c28db8a.